docs/content/gettingstarted.md

```diff
@@ -31,6 +31,13 @@ In addition, the following vendors offer free trial accounts for their Eclipse h
 | MySQL |  | &#10003; | &#10003; | 
 | RabbitMQ |  | &#10003; | &#10003; | 
 
+HawkBit Update Server docker container login credentials:
```

Comment:
Only one service feels a little bit lost in this table. I would suggest to mention the default credentials in a sentence which is enriched with some additional information. Something like:

hawkBit Update Server uses `username=admin` and `password=admin` as default login credentials. They can be overridden by the environment variables `spring.security.user.name` and `spring.security.user.password` which are defined in the corresponding default [application.properties](https://github.com/eclipse/hawkbit/blob/master/hawkbit-runtime/hawkbit-update-server/src/main/resources/application.properties).

Reply:
Replaced the table and pushed the change,.